python/sglang/srt/layers/quantization/fp8_kernel.py

```diff
@@ -33,7 +34,7 @@
     supports_custom_op,
 )
 
-_enable_jit_deepgemm = False
+enable_jit_deepgemm = False
```

Comment:
QQ why change the name here?

Reply:
Because I want to use it in the new warmup file, so theoretically speaking it should be public